docs/topics/competitive-programming.md

```diff
@@ -224,18 +223,20 @@ you can have the following list of helper input-reading functions:
 <tab title="Kotlin 1.6.0 and later" group-key="kotlin-1-6">
 
 ```kotlin
-private fun readInt() = readln().toInt() // single int
-private fun readStrings() = readln().split(" ") // list of strings
+private fun readStr() = readln() // string line
```

Comment:
Can you explain why you introduced the `readStr` function if it's just an alias for `readln`?

Reply:
Fine by me!